### DISCUSSION TITLE
Add Devices that are charging from the MacBook over USB-C to Power Flow View

### DISCUSSION THREAD

**Discussion by slepp22** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1005] [Ideas]
I love the Power Flow Diagram, however to have a complete overview of the currents that are going into the mac and the current usage it would be lovely to not only see how much it consumes by itself, but display the voltage for external charging devices (e.g. iPhone via USB-C) separately.

**Comment by MatthiasKerbl** [comment]
Hi @slepp22,

Good idea. Also, we already have a feature request open for this #1002.

Best,
Matthias
